src/proctored-exam-settings/ProctoredExamSettings.jsx

```diff
@@ -232,9 +236,11 @@ function ExamSettings(props) {
           className="btn-primary mb-3"
           data-test-id="submissionButton"
           onClick={onButtonClick}
+          disabled={submissionInProgress}
```

Comment:
There may be an accessibility issue here, because if the submit button is disabled a screen reader will not be able to "see" or focus on this element. I think using the disabled attribute would be fine if there is a way for the spinner to have a label associated with it. I don't think this is blocking, but I'd be curious to hear other people's thoughts on this.

Reply:
@alangsto Are you saying we should have an `aria-label` attribute to the in progress spinner element?

Reply:
I think that would be a good option if the button has the disabled attribute

Reply:
@alangsto Took in your feedback here.